package.json

```diff
@@ -25,8 +25,8 @@
   "version": "0.27.1-rc2",
   "private": true,
   "dependencies": {
-    "@hathor/wallet-lib": "1.0.2",
-    "@ledgerhq/hw-transport-node-hid": "6.27.1",
+    "@hathor/wallet-lib": "1.0.4",
```

Comment:
Why do we need this upgrade?

Reply:
It's not necessary, Its just that wallet-lib v1.0.3 fixed an issue with the typing of the wallet-lib, which makes my LSP more useful (autocompletes, documentation, method signatures, etc.).